include/librealsense2/dds/topics/notification/notification.idl

```diff
@@ -11,7 +11,7 @@ module librealsense
             {
                 module device
                 {
-                    struct notifications
+                    struct notification
```

Comment:
Please generate with the -typeobject flag and add the notificationTypeObject.h+cpp files

Reply:
How about `image` topic?

Do we also want it will dynamic type generated?

Reply:
If we're going the way of the sniffer knowing the structure of the Safety Camera topics (without XTypes) then I wouldn't spend too much time on it...

Reply:
We unset the exclusive ownership to match the server settings, now it's working.
Done